resin/models/device.py

```diff
@@ -686,34 +688,37 @@ def disable_device_url(self, uuid):
             endpoint=self.settings.get('pine_endpoint')
         )
 
-    def set_to_build(self, uuid, build):
+    def set_to_build(self, uuid, build_commit_hash):
```

Comment:
Note that this is a breaking change - presumably this means we'll bump to 3.0.0?

Reply:
It looks like from #57 maybe actually the old version never worked? If it definitely never worked (so we can guarantee that nobody's relying on it) then I think changing this as a patch release is fine, but if it's possible the previous method worked then we do need to mark this as major.

Reply:
...and it's only a minor nit, but I was wondering if `BuildNotFound(commit_hash)` should actually be a different exception from `BuildNotFound(id)`? Or perhaps that's such a subtle difference that it's not worth worrying about? \*shrug\*

Reply:
I know the JS SDK uses the same exception class for both, although I'd be open to changing it if there's value to doing so... The main reason for separate error classes as I understand it are:

* Catch errors for missing builds specifically (in which case I don't think you care which format was used)
* Show easily understandable errors in the console if they're not caught (ditto)

Is there a use case where you need to know which format you passed from the error itself?

Reply:
> I know the JS SDK uses the same exception class for both

Ah, in that case I'm happy for the Python SDK to do the same then :-)

Reply:
@pimterry, this function works on old pine endpoint (`ewa`) and I'm sure that we have users are using it so yeah, we will bump to v3.0.0 (will merge this PR to v3 branch and release v3 after I finish implementing the api key changes)